src/main/java/frc/robot/commands/operatorcommands/TeleFlyVarUp.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.operatorcommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import java.util.function.Supplier;
+
+import frc.robot.subsystems.flywheelvarhood.VarFlyWheel;
+
+public class TeleFlyVarUp extends CommandBase{
+    private final VarFlyWheel varFlyWheel;
+    public TeleFlyVarUp(VarFlyWheel varFlyWheel) {
+        this.varFlyWheel = varFlyWheel;
+    }
+
+    @Override
+    public void execute() {
+        varFlyWheel.extendFly();
+    }
+    
+    @Override
+    public void end(boolean interrupted) {
+        varFlyWheel.stop();
```

Comment:
Again why are we stopping wheel? Should this be a parallel command group?

Reply:
fixed